pkgs/development/python-modules/django-scim2/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonRelaxDepsHook
+
+# propagates
+, django
+, dateutil
+, scim2-filter-parser
+, gssapi
+, ldap
+, sssd
+
+# tests
+, mock
+, tox
+}:
+
+buildPythonPackage rec {
+  pname = "django-scim2";
+  version = "0.17.3";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "15five";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-5zdGPpjooiFoj+2OoglXhhKsPFB/KOHvrZWZd+1nZqU=";
+  };
+
+  buildInputs = [ sssd ];
+  propagatedBuildInputs = [
+    django
+    dateutil
+    scim2-filter-parser
+    gssapi
+    ldap
+    sssd
+  ];
+
+  nativeBuildInputs = [ pythonRelaxDepsHook ];
+  pythonRelaxDeps = [
+    "django"
+  ];
+
+  pythonImportsCheck = [ "django_scim" ];
+
+  checkInputs = [  mock tox ];
```

```suggestion
  nativeBuildInputs = [
    pythonRelaxDepsHook
  ];
  
  pythonRelaxDeps = [
    "django"
  ];

  pythonImportsCheck = [
    "django_scim"
  ];

  checkInputs = [
    mock
    unittestCheckHook
  ];
```

- Move `nativeBuildInputs` above `buildInputs`
- Expand lists to multiple lines
- Use the `unittestCheckHook` to run the tests explicitly
- Passing `tox` should not be necessary

---

`unittestCheckHook` does not work, the tests all fail. `tox` did work. What should I do?